blocks.md

```diff
@@ -0,0 +1,80 @@
+<h2>Guide to submitting plugins to the Block Directory</h2>
+
+The goal of the Block Directory is to provide a safe place for WordPress users to find and install new blocks.
+
+<h3>User Expectations</h3>
+
+* Users will have a place they can go, both within their WordPress admin and on WordPress.org, where they can download and install blocks that have been pre-vetted for major security issues. 
+* Users will be able to one-click install blocks from their admin, one at a time.
+* Blocks will appear in their block library after installation and activation.
+
+<h3>Developer Expectations</h3>
+
+* Developers will have a concrete set of requirements and guidelines to follow when writing blocks for the Block Directory
+
+<h3>Definitions</h3>
+
+First up, some clarity. For the purposes of the Block Directory, we distinguish between two types of plugins:
+1. Plugins that exist solely to distribute a block.
+1. All other plugins, including those that bundle many independent blocks, plugins that contain blocks in addition to other functionality, and plugins with no blocks at all.
+
+These guidelines apply specifically to the first type of plugin, which we’ll call here a Block Plugin. If your plugin is of the second type, then the further guidelines and restrictions do not apply to your work. All plugins, be they block-only or not, are required to comply with the Detailed Plugin Guidelines https://developer.wordpress.org/plugins/wordpress-org/detailed-plugin-guidelines/.
+
+<h3>Block Plugins and the Block Directory</h3>
+
+The Block Directory contains only Block Plugins, that is to say plugins that contain only a single independent block and a minimum of supporting code. Block plugins have the following characteristics:
+
+1. A specific type of WordPress plugin, with the same structure including a readme.txt file.
+1. Containly only blocks (typically one).
```

Comment:
The `(typically one)` statement here conflicts with the earlier statement:

> ...that is to say plugins that contain only a single independent block

and then later:

> Block plugins may contain more than one block where a clearly necessary parent/child dependency exists; for example a list block that contains list item blocks.

So I think I grasp that the guidelines are attempting to make it clear that a Block Plugin may only contain _one_ independent block with the exception of that block containing other blocks as inner blocks.  However, I still found it a bit difficult parsing that intent from what is written (and may have even parsed it wrong 😄 ).  

A suggestion:  All references regarding number of blocks should be explicit about the plugin only having one independent block and then centralize the exception in one statement/place in the doc.

With that in mind, this specific statement would be changed to "Containing only one block".

Reply:
Good point, thanks for the feedback! I've been trying to straddle a line here. Really what I'm getting at is that block plugins should be single purpose. Generally that means a single block, but there will be cases where a single purpose requires several blocks (such as the list/item example). It's hard to be precise without second-guessing what kinds of blocks and plugins people will submit. We could try to codify that as something like "irreducible single-purpose block sets" but that seems needlessly technical.

I think what we need is to reword "single independent block" in the first statement. "typically a single block" is one way. "a single block or small set of dependent blocks" is another. I'll think on it some more.